src/index.html

```diff
@@ -1,13 +1,473 @@
+                        src="./images/people/3.jpg"
+                        alt="Max Henry"
+                        class="testimonial__photo"
+                      >
+                    </div>
+  
+                    <h3 class="testimonial__name">
+                      Max Henry
+                    </h3>
+  
+                    <span class="testimonial__position">
+                      Chief Design Officer
+                    </span>
+  
+                    <p class="testimonial__about">
+                      When my lovely valley of steam rises and half-day sun is above an impermeable
+                    </p>
+                  </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="page__section contact-us" id="contacts">
+          <div class="special-wraper">
+            <h2 class="page__section-title">
+              Contact us
+            </h2>
+
+            <div class="grid grid--desktop">
+              <div class="grid__item grid__item--1-2">
+                <div class="contact-us__info">
+                  <div class="contact-us__info-title">
+                    Phone
+                  </div>
+                  <a href="tel:+1 234 5555-55-55" class="contact-us__info-link">
+                    +1 234 5555-55-55
+                  </a>
+                </div>
+
+                <div class="contact-us__info">
+                  <div class="contact-us__info-title">
+                    Email
+                  </div>
+                  <a href="mailto:hello@miami.com" class="contact-us__info-link">
+                    hello@miami.com
+                  </a>
+                </div>
+              </div>
+
+              <div class="grid__item grid__item--4-5">
+                <div class="contact-us__info contact-us__info--last">
+                  <div class="contact-us__info-title">
+                    Address
+                  </div>
+                  <a
+                    target="_blank"
+                    href="https://goo.gl/maps/x54twYkpi9LfmgRo7"
+                    class="contact-us__info-link"
+                  >
+                    400 first ave. n.
+                    <br>
+                    suite 700
+                    <br>
+                    Minneapolis, MN 55401
+                  </a>
+                </div>
+              </div>
+
+              <div class="grid__item grid__item--8-12">
+                <form action="" method="get">
+                  <input
+                    type="text"
+                    name="user-name"
+                    id="user-name"
+                    placeholder="Name"
+                    class="form-field contact-us__field"
+                  >
+                  <input
+                    type="email"
+                    name="user-email"
+                    id="user-email"
+                    placeholder="Email"
+                    class="form-field contact-us__field"
+                  >
```

Comment:
Also your page looks weird on higer resolutions, make sure to fix it https://prnt.sc/1tx96gh

Reply:
> Also your page looks weird on higer resolutions, make sure to fix it https://prnt.sc/1tx96gh

I will change it, but in my opinion it will look weirder,
because in design MAX-width 1260px and MAX paddings 120px,
now we will have huge space on sides.

Reply:
> I will change it, but in my opinion it will look weirder, because in design MAX-width 1260px and MAX paddings 120px, now we will have huge space on sides.
Yeah, but I think that these white spaces are not the best solution, you may try to come up with something more appropriate)